packages/platform-sdk-examples/src/examples/transfer.ts

```diff
@@ -0,0 +1,57 @@
+import { Contracts, Environment } from "@arkecosystem/platform-sdk-profiles";
+import { useEnvironment, useLogger } from "../helpers";
+
+export default async () => {
+	const logger = useLogger();
+	const env: Environment = await useEnvironment();
+
+	// Open profile
+	const profile: Contracts.IProfile | undefined = env.profiles().findByName("my-profile-name");
+	if (!profile) {
+		throw Error("Profile doesn't exist");
+	}
+
+	// Restore it and sync
+	await env.profiles().restore(profile, "my-password");
+	await profile.sync();
+
+	// Get read-write wallet #1
+	const wallet: Contracts.IReadWriteWallet = await profile.wallets().first();
+	const mnemonic1: string = "super secure password";
+
+	// Display profile and wallet balances
+	logger.log("Wallet", wallet.address(), "balance", wallet.balance().toHuman(2));
+
+	// Get contact #1
+	const gimli: Contracts.IContact = await profile.contacts().first();
+	const contactAddress: Contracts.IContactAddress = await gimli.addresses().first();
+	const destinationAddress: string = contactAddress.address();
+	logger.log("Gimli's Wallet", destinationAddress);
+
+	const transactionId = await wallet
+		.transaction()
+		.signTransfer({
+			from: wallet.address(),
+			sign: {
+				mnemonic: mnemonic1
```

Comment:
@faustbrian Question... given that I'm using a RW wallet from the profile, accessed the profile with the password... is there a way I don't need to provide the mnemonic again for signing this transfer?

Reply:
Oh, I thought in DW you didn't need to input that everytime. I don't use DW a lot. And when I did, I used my Nano X, so no need.

Reply:
Yeah with a ledger you don't need to enter the mnemonic directly but the mnemonic is supposed to be never stored anywhere so every time you want to sign a transaction you need to enter it. Alternatively you can use BIP38 which stores your private key as an encrypted WIF with a custom password which is faster to type.